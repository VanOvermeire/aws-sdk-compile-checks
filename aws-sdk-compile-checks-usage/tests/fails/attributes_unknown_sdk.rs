use aws_sdk_compile_checks_macro::required_props;

#[required_props(sdk = unknown)]
async fn do_call() {
    // irrelevant
}

fn main() {}
