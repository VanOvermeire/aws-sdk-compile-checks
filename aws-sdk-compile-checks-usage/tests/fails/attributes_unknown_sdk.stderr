error: some of the SDKs you specified do not exist in our list of supported SDKs: unknown
 --> tests/fails/attributes_unknown_sdk.rs:3:1
  |
3 | #[required_props(sdk = unknown)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `required_props` (in Nightly builds, run with -Z macro-backtrace for more info)
